OpenGL.Platform/Window.cs

```diff
@@ -39,7 +39,11 @@ public static class Window
         #endregion
 
         #region Create SDL Window and OpenGL Context
-        public static IntPtr window, glContext;
+        private static IntPtr window, glContext;
+        public static IntPtr windowID
```

Comment:
This looks good.  Do you mind renaming the public fields slightly to keep consistent with the rest of the project with respect to capitalization?  Private fields should be lowercase, and public properties should be uppercase.  So this could be 'Window' and glContext could become 'GlContext'.  Thanks!

Reply:
Style has been changed to better match the rest of the class